src/services/forecast.ts

```diff
@@ -28,16 +28,16 @@ export class Forecast {
   ) {}
 
   public async processForecastForBeaches(
-    beaches: Beach[]
+    beaches: Beach[],
+    orderBy: 'asc' | 'desc' = 'desc',
+    orderField = 'rating'
```

Comment:
Last thing! I promise! Can you type the `orderField`? You could use `orderField: keyof BeachForecast` and Typescript would add all the keys in that type as possible values of the orderField which is what we want right?

Reply:
sure, I'll type it.